platform-sdk/swirlds-config-impl/src/main/java/com/swirlds/config/impl/internal/ConfigurationBuilderImpl.java

```diff
@@ -88,6 +96,9 @@ public Configuration build() {
             if (initialized.get()) {
                 throw new IllegalStateException("Configuration already initialized");
             }
+            if (!properties.isEmpty()) {
+                withSource(new SimpleConfigSource(properties));
```

Comment:
We should define a specific ordinal for that source

Reply:
This now uses the following ordinal:

```
    /**
     * Ordinal for configuration passed to the platform builder via withConfig().
     */
    public static final int PLATFORM_BUILDER_ORDINAL = 50;
```